app/src/main/java/com/itssuryansh/taaveez/NotesDatabase.kt

```diff
@@ -5,7 +5,7 @@ import androidx.room.Database
 import androidx.room.Room
 import androidx.room.RoomDatabase
 
-@Database(entities = [NotesEntity::class],version=2)
+@Database(entities = [NotesEntity::class],version=2, exportSchema = false)
```

Comment:
As this app is on PlayStore, `exportSchema = false` can cause errors and crashes, which can affect user-experience. If its possible can you look for other ways, like Creating a Migration class?
Its also fine if you are unable to, will create another issue for this separately, when its merged